New optionsets

Zack-83 opened the pull request:
| Change                            | Key                                | Remarks     | Proof result      |
| --------------------------------- | ---------------------------------- | ----------- | ----------------- |
| New optionset                     | access_permissions                 | 3 options   | &check; discussed |
| New optionset                     | access_type                        | 4 options   | &check; discussed |
| New optionset                     | backup_frequency                   | 5 options   | &check; approved  |
| New optionset                     | file_type                          | 10 options  | &check; renamed   |
| New optionset                     | embargo_why                        | 3 options   | &check; updated   |
| New optionset                     | licence                            | 3 options   | &check; updated   |
| New optionset                     | metadata                           | 9 options   | &check; approved  |
| ~New optionset~                   | ~objectives~                       | ~6 options~ | &cross; removed   |
| New optionset                     | preservation_responsible           | 6 options   | &check; approved  |
| ~New optionset~                   | ~purpose~                          | ~4 options~ | &cross; removed   |
| New optionset                     | repository_certification           | 11 options  | &check; updated   |
| New optionset                     | security_options                   | 3 options   | &check; approved  |
| ~New optionset~                   | ~sharing~                          | ~4 options~ | &cross; removed   |
| New optionset                     | storage_support                    | 11 options  | &check; approved  |
| ~New optionset~                   | ~usage_objectives~                 | ~6 options~ | &cross; removed   |
| New optionset                     | usage_software                     | 4 options   | &check; approved  |
| New optionset                     | vocabulary_physics                 | 5 options   | &check; renamed   |
| New single option                 | dataset_sharing_options            |             | &check; approved  |
| New single option                 | preservation_repository_options    |             | &check; approved  |
| ~Wording of some options changed~ | ~dataset_interoperability_options~ |             | &cross; reverted  |
| Wording of some options changed   | dataset_size_options               |             |                   |
| Wording of some options changed   | preservation_motivation_options    |             |                   |
| Wording of some options changed   | preservation_repository_options    |             |                   |
| Ordering of options changed       | …                                  |             |                   |
| French + Italian translation      |                                    |             |                   |

Freund-J commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -881,10 +2328,24 @@
+		<text lang="it">Sì, internamente con tutti, purché non rendano pubblico il codice</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/dataset_sharing_options/anonym">
```
Ich finde die neue Option sehr gut. Bei der Veröffentlichung sollten wir darauf achten, dass wir im Changelog auf neue Optionen in bestehenden Optionensets besonders hinweisen, da sie dann auch in bestehenden Fragenkatalogen angezeigt werden.

Freund-J commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1378,18 +2875,20 @@
 		<text lang="en">Identifiers</text>
 		<text lang="de">Identifikatoren</text>
 		<text lang="fr">Identifiants</text>
+		<text lang="it">Identificatori</text>
 		<additional_input>False</additional_input>
 	</option>
-	<option dc:uri="https://rdmorganiser.github.io/terms/options/mandatory_metadata_options/108">
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/mandatory_metadata_options/110">
```
Kann es bei der Umbenennung von bestehenden Optionen theoretisch zu Problemen in Ansichten kommen? (falls in einer Ansicht auf diese Option verwiesen wird)

Freund-J commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1765,9 +3304,10 @@
 		<dc:comment/>
 		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/preservation_repository_options"/>
 		<order>4</order>
-		<text lang="en">Has not yet been decided</text>
-		<text lang="de">Wurde noch nicht entschieden</text>
-		<text lang="fr">N'a pas encore été décidé</text>
+		<text lang="en">Will be discussed in a project meeting</text>
+		<text lang="de">Wird in einem Projekttreffen besprochen</text>
+		<text lang="fr">Va être discuté dans une réunion du projet</text>
+		<text lang="it">Sarà oggetto di discussione in una riunione di progetto</text>
```
Ich finde die neue, positive Formulierung auf jeden Fall besser. Falls die Optionen auch in bereits ausgefüllten Katalogen nach dem Import geändert werden, müssten wir darauf aber im Changelog deutlich hinweisen. Ich könnte mir vorstellen, dass es ansonsten für die Wissenschaftler nicht nachvollziehbar ist, dass sich die Formulierung der von ihnen ausgewählten Antworten geändert hat.

torsten-rathmann replied:
Das sehe ich auch so. Gibt es schon eine Versionierung oder ändern wir hier mit einem administrativen Federstrich die Antworten der Forschenden?

Zack-83 replied:
Kurze Rückmeldung von Johannes: eine positive Formulierung ist vorzuziehen, solange die Bedeutung wirklich deckungsgleich mit der vorherigen ist. Strikt gesagt ist "wie werden daran denken" eine unterschiedliche Option als "wir haben noch nicht daran gedacht". Fällt euch eine positive Umformulierung ein?

jochenklar replied:
Wird noch entschieden / To be decided

torsten-rathmann commented:
Do we expect from future catalog authors usage of all existing option sets in their catalog?

torsten-rathmann commented:
**New optionsets access_permissions and access_type:** 
OK but please check, if the already existing optionset dataset_sharing_options could be extended instead

**New optionset backup_frequency**
OK

**New optionset data_type**
For which question is this optionset designed? I think it neither matches "What kind of dataset is it?" nor "Which file formats are used?" really good, e,g. the given examples in the help for question "What kind of dataset is it?" can't be found in the set. Additionally, for both questions an option set would be constricting. For example, detailed answers as 

- Anonymised X-ray images and survey data from patients
- X-ray images are in the format DICOM 3 (Digital Imaging and Communications in Medicine), survey data in PDF-A

can't no longer be given here. I know that for maDMP such optionset must be designed, nevertheless they would restrict researchers and information will be got lost. Jochen's choice to have a text area with these questions is right in my opinion.

**New optionset embargo_why**
OK. An option "finishing of a bachelor/master/doctor thesis" should be added, maybe in a subsequent pull-request?

**New optionset licence**
Good idea! The Open Data Commons licenses should be added because they are explicitly mentioned in Horizon Europe document af_he-ria-ia_en.pdf. Please open additional input field "<additional_input>True</additional_input>", since CC licenses exist in different versions and languages. For example, in DSpace 6 repositories only CC BY in version 3 can be selected, not the newest version 4. Some research projects need 4, since it better protects data base rights.

**New optionset metadata**
OK

**New optionset objectives**
What is the meaning of "Objective 1" to "Objective 5"?

**New optionset preservation_responsible**
OK

**New optionset purpose**
Very special, really necessary here?

**New optionset repository_certification**
- nestor is missing
- DINI is a certificate for Open Access publication services (document services), not for data repositories.
- DSA and WDS certs are outdated and now involved into Core TrustSeal
- I would list ISO 16363 instead of DIN 31644 here

torsten-rathmann commented:
**New optionset security_options**
OK

**New optionset sharing**
In the generic rdmo catalog, most questions are answered for each dataset separately ("is collection" is checked in the definition of the question). That is the case in all questions using a project/dataset attribute as I know. Unless we want to change this, the new optionset doesn't make sense or shall a dataset be devided into smaller pieces with download allowed/not allowed? I don't think that such a principle change would be a win and most repositories don't allow to set different access right for parts of a dataset.

**New optionset storage_support**
OK

**New optionset usage_objectives**
OK but this will lead users to use keywords, since the additional_input field is small. Is this intended?

**New optionset usage_software**
OK

**New optionset vocabulary**
Seems to be very special, really necessary here?

torsten-rathmann commented:
**Wording of some options changed in optionset dataset_interoperability_options**
Change to plural "datasets" dosn't match the use of this optionset, since question "Is this dataset interoperable, i.e. allowing data exchange and re-use between researchers, institutions, organisations, countries etc.?" is answered for each dataset separately.

**New single option 138 in optionset preservation_repository_options**
OK

Zack-83 commented:
@torsten-rathmann Many thanks for your suggestions. I tried to address most of them. Here my comments:

> **New optionsets access_permissions and access_type:**
> OK but please check, if the already existing optionset dataset_sharing_options could be extended instead
> 
I would do it willingly, but the main problem is that the options in `dataset_sharing_options` are very verbose, so that it becomes difficult to build a text answer around them. For instance if the question sounds "**Who** can access the data?" or "**For whom** will the data be accessible?" the answer cannot be "externally **with** everyone".  
One possible compromise would be to remove simplify the wording by removing everything between "Yes, ..." and "with". But would it be back-compatible?

> **New optionset data_type**
> For which question is this optionset designed? I think it neither matches "What kind of dataset is it?" nor "Which file formats are used?" really good, e,g. the given examples in the help for question "What kind of dataset is it?" can't be found in the set. Additionally, for both questions an option set would be constricting. For example, detailed answers as
> 
>     * Anonymised X-ray images and survey data from patients
> 
>     * X-ray images are in the format DICOM 3 (Digital Imaging and Communications in Medicine), survey data in PDF-A
> 
> 
> can't no longer be given here. I know that for maDMP such optionset must be designed, nevertheless they would restrict researchers and information will be got lost. Jochen's choice to have a text area with these questions is right in my opinion.
> 
I used it for the second half of the H2020 question "What types and **formats** of data will the project generate/collect?".

> **New optionset embargo_why**
> - [x] OK. An option "finishing of a bachelor/master/doctor thesis" should be added, maybe in a subsequent pull-request?
> 
Done. For German I adopted the wording "Anfertigung einer Abschlussarbeit (Bachelor-, Master- oder Doktor-)"

> **New optionset licence**
> - [x] Good idea! The Open Data Commons licenses should be added because they are explicitly mentioned in Horizon Europe document af_he-ria-ia_en.pdf. Please open additional input field "<additional_input>True</additional_input>", since CC licenses exist in different versions and languages. For example, in DSpace 6 repositories only CC BY in version 3 can be selected, not the newest version 4. Some research projects need 4, since it better protects data base rights.
> 
Done.

> **New optionset objectives**
> What is the meaning of "Objective 1" to "Objective 5"?
> 
In H2020 the first question reads "What is the purpose of the data collection/generation **and its relation to the objectives of the project**?" so a connection to the project's objectives is requested.

> **New optionset purpose**
> Very special, really necessary here?
> 
It also relates to the first question in H2020. I admit that my options are quite metrology-specific. If it is too difficult to design broad enough answers, i can drop this optionset.

> **New optionset repository_certification**
>
>     * nestor is missing
>
>     * DINI is a certificate for Open Access publication services (document services), not for data repositories.
> 
>     * DSA and WDS certs are outdated and now involved into Core TrustSeal
> 
>     * I would list ISO 16363 instead of DIN 31644 here
>
I added Nestor and ISO 16363 and dropped the three outdated ones. I would keep anyway DIN 31644 - or is it also outdated?

Zack-83 commented:
> **Wording of some options changed in optionset dataset_interoperability_options**
> - [x] Change to plural "datasets" dosn't match the use of this optionset, since question "Is this dataset interoperable, i.e. allowing data exchange and re-use between researchers, institutions, organisations, countries etc.?" is answered for each dataset separately.
> 
I reverted it. I had changed to plural because so far I was dealing only with "initial" DMPs, where it is not clear which and how many dataset will be generated, and therefore I let the questions be answered without reference to single datasets.

Zack-83 commented:
> **New optionset sharing**
> In the generic rdmo catalog, most questions are answered for each dataset separately ("is collection" is checked in the definition of the question). That is the case in all questions using a project/dataset attribute as I know. Unless we want to change this, the new optionset doesn't make sense or shall a dataset be devided into smaller pieces with download allowed/not allowed? I don't think that such a principle change would be a win and most repositories don't allow to set different access right for parts of a dataset.
> 
This is also a consequence of my "initial DMP bias".  
We could discuss in the community, if anybody else is interested in optiond for initial DMPs. If no, I will drop it.

> **New optionset usage_objectives**
> OK but this will lead users to use keywords, since the additional_input field is small. Is this intended?
> 
No. This is also related to H2020 question 1 (which for "my" funding call is divided into two questions, therefore requires more effort). I expect an answer such as
* _Objective 3: Data are used to develop an experimental technique for on-demand single-electron wave packet interferometry for the sensing of local magnetic and electric fields with high time resolution._

> **New optionset vocabulary**
> Seems to be very special, really necessary here?
>
I agree; this is physics-specific. If it can be still of general relevance, we can rename it `vocabulary_physics`, otherwise it can be dropped.

torsten-rathmann commented:
> > **New optionsets access_permissions and access_type:**
> > OK but please check, if the already existing optionset dataset_sharing_options could be extended instead
> > 
> I would do it willingly, but the main problem is that the options in `dataset_sharing_options` are very verbose, so that it becomes difficult to build a text answer around them. For instance if the question sounds "**Who** can access the data?" or "**For whom** will the data be accessible?" the answer cannot be "externally **with** everyone".  
> One possible compromise would be to remove simplify the wording by removing everything between "Yes, ..." and "with". But would it be back-compatible?
> 
Perhaps you can help me better understand your concern if you state the question(s) with which the two new optionsets should be used.

torsten-rathmann commented:
> > **New optionset data_type**
> > For which question is this optionset designed? I think it neither matches "What kind of dataset is it?" nor "Which file formats are used?" really good, e,g. the given examples in the help for question "What kind of dataset is it?" can't be found in the set. Additionally, for both questions an option set would be constricting. For example, detailed answers as
> > 
> >     * Anonymised X-ray images and survey data from patients
> > 
> >     * X-ray images are in the format DICOM 3 (Digital Imaging and Communications in Medicine), survey data in PDF-A
> > 
> > 
> > can't no longer be given here. I know that for maDMP such optionset must be designed, nevertheless they would restrict researchers and information will be got lost. Jochen's choice to have a text area with these questions is right in my opinion.
> > 
> I used it for the second half of the H2020 question "What types and **formats** of data will the project generate/collect?".
I see, thanks! Nonetheless, even more details should be given in the answer if the question queries data types **and** formats. It isn't possible to answer e.g. "Digitization of enquete commission's proceedings and reports, TIFF images with METS/MODS metadata files, full texts in XML/TEI I5" within the answer options of MIME.

torsten-rathmann commented:
> > **New optionset purpose**
> > Very special, really necessary here?
> > 
> It also relates to the first question in H2020. I admit that my options are quite metrology-specific. If it is too difficult to design broad enough answers, i can drop this optionset.

It would be really difficult to provide similar options for all subjects. So, I also think it's better to drop it.

torsten-rathmann commented:
> > **New optionset repository_certification**
> >
> >     * nestor is missing
> >
> >     * DINI is a certificate for Open Access publication services (document services), not for data repositories.
> > 
> >     * DSA and WDS certs are outdated and now involved into Core TrustSeal
> > 
> >     * I would list ISO 16363 instead of DIN 31644 here
> >
> I added Nestor and ISO 16363 and dropped the three outdated ones. I would keep anyway DIN 31644 - or is it also outdated?

DIN 31644 is still in use. The nestor seal is based on DIN 31644. Nevertheless, I doubt that nestor users know that and correctly check DIN 31644 too. Maybe it's best to drop both, DIN 31644 and ISO 16363. In case users really need it, they can use option "Other". The same option could be used in those special cases, the repository still uses a DSA or WDS cert or the research data are documents itself and the repo has a DINI cert.

By the way, nestor ist written in small letters.

torsten-rathmann commented:
> > **New optionset sharing**
> > In the generic rdmo catalog, most questions are answered for each dataset separately ("is collection" is checked in the definition of the question). That is the case in all questions using a project/dataset attribute as I know. Unless we want to change this, the new optionset doesn't make sense or shall a dataset be devided into smaller pieces with download allowed/not allowed? I don't think that such a principle change would be a win and most repositories don't allow to set different access right for parts of a dataset.
> > 
> This is also a consequence of my "initial DMP bias".  
> We could discuss in the community, if anybody else is interested in optiond for initial DMPs. If no, I will drop it.

Initial DMPs are also in my focus. We should discuss this topic first and maybe as a block since I also want to contribute to this field. Maybe it's best to drop it now and discuss it first.

torsten-rathmann commented:
> > **New optionset usage_objectives**
> > OK but this will lead users to use keywords, since the additional_input field is small. Is this intended?
> > 
> No. This is also related to H2020 question 1 (which for "my" funding call is divided into two questions, therefore requires more effort). I expect an answer such as
> * _Objective 3: Data are used to develop an experimental technique for on-demand single-electron wave packet interferometry for the sensing of local magnetic and electric fields with high time resolution._

OK

torsten-rathmann commented:
**New optionset license**
Maybe there has been a misunderstanding in the last commit. Open Data Commons is an institution. We have to add single ODC licenses, i.e. the

- [Open Data Commons Attribution License (ODC-By)](https://opendatacommons.org/licenses/by/1-0/) (similar to CC BY)
- [Open Database License (ODC-ODbL)](https://opendatacommons.org/licenses/odbl/1-0/) (similar to CC BY-SA)

What's about providing links to the legal texts, i.e.
[CC BY](https://creativecommons.org/licenses/by/4.0/legalcode.de)
[CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/legalcode.de)

torsten-rathmann commented:
> > **New optionset vocabulary**
> > Seems to be very special, really necessary here?
> >
> I agree; this is physics-specific. If it can be still of general relevance, we can rename it `vocabulary_physics`, otherwise it can be dropped.

`vocabulary_physics` would be OK for me. We should ask the development group if a mechanism for switching between an overview (optionset names only) and full display of all options could be implemented in the web surface of the RDMO Management.

Zack-83 commented:
> We should ask the development group if a mechanism for switching between an overview (optionset names only) and full display of all options could be implemented in the web surface of the RDMO Management.

@jochenklar @triole

jochenklar commented:
@Zack-83 Makes sense, I created an issue https://github.com/rdmorganiser/rdmo/issues/329

torsten-rathmann commented:
Nearly done! The optionsets purpose and sharing should be dropped.

**Optionset purpose** is institution-specific, not even physics-specific. For example, which option shall be used for determination of the value of a constant which cannot be measured with high precision but only the order of magnitude or even an upper limit? Is dark matter a material, a natural phenomenon or neither one? And we will run into problems if we try to extend the optionset for all subjects.

**Optionset sharing** and "initial DMPs" would lead to catalogs without dataset collection and DMPs without dataset specification. In case a user wants to have a new version of the DMP later and tries to switch to another catalog using dataset collections, the DMP cannot be edited unless the catalog switch has been revoked. The answers given in the initial DMP without dataset collection were'nt overtaken too. This I have tested with RDMO 1.5.5, catalog rdmo and questionset "Date collection". So I don't think we should provide optionsets for catalogs without dataset collection now. Catalog change must work as long as it is offered to normal users.

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1385 @@
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/purpose/constant">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>constant</key>
+		<path>purpose/constant</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/purpose"/>
+		<order>0</order>
+		<text lang="en">Determination of the value of a constant with high precision</text>
+		<text lang="de">Bestimmung des Werts einer Naturkonstante mit höher Präzizion</text>
+		<text lang="fr">Détermination de la valeur d'une constante de la nature avec une haute precision</text>
+		<text lang="it">Determinazione del valore di una costante della natura con alta precisione</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/purpose/phenomenon">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>phenomenon</key>
+		<path>purpose/phenomenon</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/purpose"/>
+		<order>0</order>
+		<text lang="en">Description of a natural phenomenon</text>
+		<text lang="de">Beschreibung eines Naturphänomens</text>
+		<text lang="fr">Description d’un phénomène naturel</text>
+		<text lang="it">Descrizione di un fenomeno naturale</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/purpose/other">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>other</key>
+		<path>purpose/other</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/purpose"/>
+		<order>0</order>
+		<text lang="en">Other</text>
+		<text lang="de">Sonstiges</text>
+		<text lang="fr">Autre</text>
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
+	</option>
```
```suggestion
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1326 @@
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/sharing/anonym">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>anonym</key>
+		<path>sharing/anonym</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/sharing"/>
+		<order>2</order>
+		<text lang="en">Only the anonymised data</text>
+		<text lang="de">Nur die anonymisierten Daten</text>
+		<text lang="fr">Seulement les données anonymisées</text>
+		<text lang="it">Solo i dati anonimizzati</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/sharing/partly">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>partly</key>
+		<path>sharing/partly</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/sharing"/>
+		<order>3</order>
+		<text lang="en">Only part of the data</text>
+		<text lang="de">Nur ein Teil der Daten</text>
+		<text lang="fr">Seulement une partie des données</text>
+		<text lang="it">Solo una parte dei dati</text>
+		<additional_input>True</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/sharing/none">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>none</key>
+		<path>sharing/none</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/sharing"/>
+		<order>4</order>
+		<text lang="en">No data, because</text>
+		<text lang="de">Keine Daten, weil</text>
+		<text lang="fr">Pas de données, car</text>
+		<text lang="it">Niente dati, perché</text>
+		<additional_input>True</additional_input>
+	</option>
```
```suggestion
```

jmfrenzel reviewed (changes requested):
I have some general comments:

First of all this PR is very extensive and not easy to digest. 
One weakness is that in most cases the context of the option sets is not very clear. The PR would benefit from providing comments (comments field).  Also, the clarity of the option texts could be improved for the rdmo user, e.g. by providing additional option texts, but also for catalog developers, e.g. by adding comments. 

New keys tend to be cryptic, e.g. two letters. If this is done, then more context should be provided in these cases in particular. 

* I have added some questions an comments inline.
* It would be great, but not required, if we could get a review for the translations.

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1236,9 +2604,10 @@
 		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/infrastructure_resources"/>
 		<order>1</order>
-		<text lang="en">Infrastructure resources of the usual workplace equipment are sufficient.</text>
+		<text lang="en">Infrastructure resources of	the usual workplace equipment are sufficient.</text>
```
```suggestion
		<text lang="en">Infrastructure resources of the usual workplace equipment are sufficient.</text>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1461,9 +2846,10 @@
 		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/metadata_standards"/>
 		<order>1</order>
-		<text lang="en">Discipline-specific standards, classifications etc. are used</text>
+		<text lang="en">Discipline-specific	standards, classifications etc. are used</text>
```
```suggestion
		<text lang="en">Discipline-specific standards, classifications etc. are used</text>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+	<optionset dc:uri="https://rdmorganiser.github.io/terms/options/access_type">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>access_type</key>
+		<dc:comment>von Zenodo</dc:comment>
```
```suggestion
		<dc:comment>These options are presented by Zenodo (and possibly other repositories) when submitting a dataset for publication</dc:comment>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<text lang="de">Sonstiges</text>
+		<text lang="fr">Autre</text>
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
```
When the user answers "Other", it would be good to offer him/her a free text field to write more. In this case it's trivial (we don't know how many objectives a project has) but for other option sets it can be relevant

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives/obj4">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>obj4</key>
+		<path>usage_objectives/obj4</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives"/>
+		<order>4</order>
+		<text lang="en">Objective 4</text>
+		<text lang="de">Ziel 4 (Objective 4)</text>
+		<text lang="fr">Objectif 4</text>
+		<text lang="it">Obiettivo 4</text>
+		<additional_input>True</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives/obj5">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>obj5</key>
+		<path>usage_objectives/obj5</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives"/>
+		<order>5</order>
+		<text lang="en">Objective 5</text>
+		<text lang="de">Ziel 5 (Objective 5)</text>
+		<text lang="fr">Objectif 5</text>
+		<text lang="it">Obiettivo 5</text>
+		<additional_input>True</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives/other">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>other</key>
+		<path>usage_objectives/other</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives"/>
+		<order>6</order>
+		<text lang="en">Other</text>
+		<text lang="de">Sonstiges</text>
+		<text lang="fr">Autre</text>
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
+	</option>
```
```suggestion
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -2381,9 +3850,10 @@
 		<order>1</order>
-		<text lang="en">Law</text>
+		<text lang="en">Law: "Recht des Herstellers eines Tonträgers"</text>
 		<text lang="de">Recht des Herstellers eines Tonträgers</text>
```
Ich stimme zu. Wer hat diese Gesetze aufgeführt und wo sind sie zu finden? Meine entsprechende Issue #26 bekam soweit keine Antwort.

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
+	</option>
+	<optionset dc:uri="https://rdmorganiser.github.io/terms/options/objectives">
```
It is about the objectives of the project. It is required for the first H2020 question: "What is the purpose of the data collection/generation **and its relation to the objectives of the project**?"

Zack-83 replied:
Done. I admit that the numbers 1 to 5 are arbitrary. Maybe could we allow the user define many objectives similarly to the many partners and to the many datasets. Or maybe it is too much work for a not so relevant point.

jmfrenzel replied:
OK, I understand. But now I don't understand why we need an option set at all. Are you planning to add conditions or tasks which are then triggered the the options? If not,  I suggest to use a simple text form which gives the users enough freedom to list their objectives.

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>title</key>
+		<path>metadata/title</path>
+		<dc:comment/>
```
The paths are always reported like that: `<optionset>/<option>`

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
+	</option>
+	<optionset dc:uri="https://rdmorganiser.github.io/terms/options/usage_objectives">
```
Agree, I removed it

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>identifier</key>
+		<path>metadata/identifier</path>
+		<dc:comment/>
```
It is a rough overview of relevant information which will be included in the metadata. "Identifier" and "Title" came from the guidelines of our funders, but now that you point to them I recognize that they sound quite stupid. I will remove them.

jmfrenzel replied:
ok. But what about to add the context of the option set to <dc:comment/>?

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<order>0</order>
+		<conditions/>
+	</optionset>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/metadata/identifier">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>identifier</key>
+		<path>metadata/identifier</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/metadata"/>
+		<order>1</order>
+		<text lang="en">identifier (PID)</text>
+		<text lang="de">Identifikator (PID)</text>
+		<text lang="fr">identifiant (PID)</text>
+		<text lang="it">identificatore (PID)</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/metadata/title">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>title</key>
+		<path>metadata/title</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/metadata"/>
+		<order>2</order>
+		<text lang="en">title</text>
+		<text lang="de">Titel</text>
+		<text lang="fr">titre</text>
+		<text lang="it">titolo</text>
+		<additional_input>False</additional_input>
+	</option>
```
```suggestion
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1182 @@
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>other</key>
+		<path>metadata/other</path>
+		<dc:comment/>
```
```suggestion
		<dc:comment>This optionset just lists the objects of the project</dc:comment>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/access_type"/>
+		<order>4</order>
+		<text lang="en">Closed access</text>
+		<text lang="de">geschlossener Zugang</text>
```
```suggestion
		<text lang="de">Geschlossener Zugang</text>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1267 @@
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/objectives/obj4">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>obj4</key>
+		<path>objectives/obj4</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/objectives"/>
+		<order>4</order>
+		<text lang="en">Objective 4</text>
+		<text lang="de">Ziel 4 (Objective 4)</text>
+		<text lang="fr">Objectif 4</text>
+		<text lang="it">Obiettivo 4</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/objectives/obj5">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>obj5</key>
+		<path>objectives/obj5</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/objectives"/>
+		<order>5</order>
+		<text lang="en">Objective 5</text>
+		<text lang="de">Ziel 5 (Objective 5)</text>
+		<text lang="fr">Objectif 5</text>
+		<text lang="it">Obiettivo 5</text>
+		<additional_input>False</additional_input>
+	</option>
+	<option dc:uri="https://rdmorganiser.github.io/terms/options/objectives/other">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>other</key>
+		<path>objectives/other</path>
+		<dc:comment/>
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/objectives"/>
+		<order>6</order>
+		<text lang="en">Other</text>
+		<text lang="de">Sonstiges</text>
+		<text lang="fr">Autre</text>
+		<text lang="it">Altro</text>
+		<additional_input>True</additional_input>
+	</option>
```
```suggestion
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -1,5 +1,1071 @@
+		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/file_type"/>
+		<order>0</order>
+		<text lang="en">statistical analysis</text>
+		<text lang="de">Statistische Eingabe</text>
```
```suggestion
		<text lang="de">Statistische Auswertung</text>
```

Zack-83 commented on rdmorganiser/options/rdmo.xml:
```diff
@@ -881,10 +2014,24 @@
+		<order>2</order>
+		<text lang="en">Yes, externally for everyone, after removal of the person-related information</text>
+		<text lang="de">Ja, extern für alle, nach Entfernung der personenbezogenen Informationen</text>
+		<text lang="fr">Oui, en externe pour tout le monte, après l'effacement des données personnelles</text>
```
```suggestion
		<text lang="fr">Oui, en externe pour tout le monde, après l'effacement des données personnelles</text>
```

jmfrenzel commented:
@Zack-83 Seems that work from this PR went into #105

Zack-83 commented:
> 
> 
> @Zack-83 Seems that work from this PR went into #105

Exactly. I tried to update this PR by merging _from_ `dev`, but GitHub understood me wrong and merged the other way. Therefore I had to revert it as fast as possible and to create a new, hopefully cleaner, merge #105. Please consider this merge and the corresponding branch `Zack-83-patch-2` as dead.
